src/mlpack/methods/reinforcement_learning/sac.hpp

```diff
@@ -0,0 +1,199 @@
+/**
+ * @file methods/reinforcement_learning/sac.hpp
+ * @author Nishant Kumar
+ *
+ * This file is the definition of SAC class, which implements the
+ * soft actor-critic algorithm.
```

Comment:
```suggestion
 * Soft Actor-Critic algorithm.
```

Just to be consistent.